src/_posts/databases/postgresql/2000-01-01-managing.md

```diff
@@ -0,0 +1,220 @@
+2. Click on the application for which you want to scale the Scalingo for
+   PostgreSQL® addon
+3. Click on the **Resources** tab
+4. Locate the **Addons** block and click on the **...** button
+5. From the dropdown menu, select **Change plan**
+6. Select the new plan
+7. Click the **Finish** button
+8. Validate by clicking the **Confirm plan change** button
+
+### Using the Command Line
+
+1. Make sure you have correctly [setup the Scalingo Command Line tool]({% post_url platform/cli/2000-01-01-start %})
+2. From the command line, list the plans available for `postgresql`:
+   ```bash
+   scalingo addons-plans postgresql
+   ```
+   The output should look like this:
+   ```bash
+   +----------------------------+---------------+
+   |             ID             |     NAME      |
+   +----------------------------+---------------+
+   | postgresql-sandbox         | Sandbox       |
+   | postgresql-starter-512     | Starter 512M  |
+   | postgresql-starter-1024    | Starter 1G    |
+   ...
+   ```
+3. Locate the `ID` corresponding to the plan you want to scale to (for example
+   `postgresql-business-1024`)
+4. List the addons attached to your application:
+   ```bash
+   scalingo --app my-app addons
+   ```
+   The output should look like this:
+   ```bash
+   +------------+-----------------------------------------+------------------------+---------+
+   |   ADDON    |                   ID                    |          PLAN          | STATUS  |
+   +------------+-----------------------------------------+------------------------+---------+
+   | PostgreSQL | ad-871546ad-943a-4929-9770-ec7c839d65f5 | postgresql-starter-512 | running |
+   ...
+   ```
+5. Locate the `ID` corresponding to the addon you want to scale
+6. Change plan using the `addons-upgrade` sub-command:
+   ```bash
+   scalingo --app my-app addons-upgrade <addon_ID> <plan_ID>
+   ```
+   The output should look like this:
+   ```bash
+   -----> Addon ad-d0aa540a-5ed2-41f8-8bbe-91e3aff6623b has been upgraded
+          Message from addon provider: Database plan is being changed
+   ```
+
+### Using the Terraform Provider
+
+1. Update the `plan` property of the corresponding Resource block in your
+   Terraform file to scale the addon:
+   ```tf
+   resource "scalingo_addon" "my-db" {
+     provider_id = "postgresql"
+     plan = "postgresql-business-1024"
+     app = "${scalingo_app.my-app.id}"
+   }
+   ```
+   In this example, we switch the `my-db` resource attached to the `my-app`
+   application to a PostgreSQL Business 1024 addon.
+2. Run `terraform plan` and check if the result looks good
+3. If so, run `terraform apply`
+
+
+## Upgrading
+
+Upgrading your Scalingo for PostgreSQL® addon consists in changing the database
+version for a newer one.
+
+When the database vendor releases a new version of your database engine, we try
+to make this new version available as soon as possible. Upgrading to this new
```

Comment:
"we try to make this new version available as soon as possible": that is not correct. We usually take our time to make these available. Especially for new major versions as we need the underlying tools to be compatible with this new version.

Reply:
For me this time is included in the "as possible". But I get what you mean and try to come up with a more appropriate wording.

Reply:
Done in https://github.com/Scalingo/documentation/pull/2389/commits/09d6b74904f24b4c529e82718af18aff9b06102d